### PR TITLE
Fixed font-family redundancies on style.css

### PR DESCRIPTION
**Pull request by wesleyParriott**
I changed all `font: Xpx lato, sans-serif` to `font-size: Xpx`

### CODE REVIEW

File: client/css/style.css [hunk 1]
```diff
@@ -128,7 +128,7 @@ button,
 	border-radius: 3px;
 	color: #84ce88;
 	display: inline-block;
-	font: bold 12px Lato, sans-serif;
+	font: bold 12px;
```
Comment (maxpoulin64) [inline client/css/style.css]: This is not a valid declaration. It is ignored entirely by Chrome. I think you will have to either use `font: bold 12px inherit` or split it into two lines.

**Comment by maxpoulin64** [comment]
Looks good to me, apart the comment I just made! 👏 When you fix that, please also reword your commit message to reflect the changes, something like `Remove duplicate font declarations`. Just so we know just from the commit message what each commit does without having to check the changes. This makes it quicker to trace back bugs by just checking the history :)

For reference, this fixes #138. If you fix other issues in the future please say something like `Fixes #XXX` in the description of the PR so we don't have to search which issue (thankfully I remembered seeing it this morning so I managed). This also makes it so GitHub closes the issue automatically on merge.

**Comment by wesleyParriott** [comment]
I'm new to this tool (just started today) so I have a rudimentary questions. 
Since I linked it to a fork and I commit a change to the font property will the pull request update automatically?

**Comment by maxpoulin64** [comment]
Yep, the PR always track your fork's branch. Welcome to the world of git/github and our project! If you need assistance with anything feel absolutely free to ping me on IRC or email me!

Moving to this PR, you will have to squash those two commits together so when we merge the PR we don't push all those intermediate commits to master. To do that, you will want to rewrite the history with the `git rebase` command. We recommend to always rebase to our latest master, so:

* Add add our repo to your local git clone, alongside of your `origin`: `git remote add lounge https://github.com/thelounge/lounge.git`
* Download that repo: `git fetch lounge`
* Now you are ready to rebase: `git rebase -i lounge/master`

The last command should open your text editor, with your two commits and a few instructions. You will want to `edit` your first commit, and `squash` the second one. Save and exit. Git will then revert to your first commit, from which you can `git commit --amend` to change the commit message as I asked. Then you can `git rebase --continue` and git ahould squash the second commit in the first one as we told it to. And finally, you can force push those changes back to your github with `git push -f origin <your-branch>`. Note that rewriting history is *only* recommended on topic branches like that. It's heavily recommended to avoid doing that on branches other people are working on at the same time.

If you need more information, GitHub has some great documentation on that topic. Hopefully this gets you going, if not feel free to ask for more information :)

**Comment by xPaw** [comment]
Good stuff. Could you do the the same in themes?

**Comment by wesleyParriott** [comment]
I'm on it as soon as I get home from work at 9pm.

**Comment by astorije** [comment]
That's great stuff indeed, @wesleyParriott! 👍

**Comment by astorije** [comment]
Could you follow @maxpoulin64's instructions (https://github.com/thelounge/lounge/pull/377#issuecomment-223739908) to squash all your commits into one before we can ship this please? Thanks!

**Comment by astorije** [comment]
Hey @wesleyParriott, any update on this? Also, GitHub will not let us merge until you fix the conflicts (lots have changed in 2 months as you can expect 😅), so if you could rebase on master and fix them, that'd be awesome.

Thanks!

**Comment by astorije** [comment]
Closing in favor of #562.

**Pull request closed without merging**